include/nlohmann/json.hpp

```diff
@@ -5637,13 +5637,14 @@ class basic_json
       their stored values are the same according to their respective
       `operator==`.
     - Integer and floating-point numbers are automatically converted before
-      comparison. Note than two NaN values are always treated as unequal.
+      comparison.
+    - Two NaN values are always treated as unequal.
```

Comment:
I would like to keep this in one item, as NaN comparison is related to the previous sentence.